future/rsfmri-analysis/rsfmri-analysis.md

```diff
@@ -8,4 +8,124 @@ permalink: /docs/rsfmri-analysis
 
 # Resting State fMRI Analysis
 
+For resting state analysis, you can use XCP Imaging Pipeline. It's free and open-sourced.
 
+It takes the FMRIPREP output as its input and ... (XXX WILL BE EDITED XXX)
```

Comment:
Add a note that the fMRIPrep outputs must be from a sufficiently recent version (I think mid 2019ish)—someone had an issue where they had old fmriprep data with different filenames.